website/docs/users-sources/sources/directory-sync/active-directory/index.md

```diff
@@ -60,7 +60,8 @@ Additional settings that might need to be adjusted based on the setup of your do
 - Addition User/Group DN: Additional DN which is _prepended_ to your Base DN configured above to limit the scope of synchronization for Users and Groups
 - User object filter: Which objects should be considered users. For Active Directory set it to `(&(objectClass=user)(!(objectClass=computer)))` to exclude Computer accounts.
 - Group object filter: Which objects should be considered groups.
-- Group membership field: Which user field saves the group membership
+- Lookup using a user attribute: Acquire group membership from a User object attribute (`memberOf`) instead of a Group attribute (`member`). This works with directories with nested groups memberships (Active Directory, RedHat IDM/FreeIPA), using `memberOf:1.2.840.113556.1.4.1941:` as the group membership field.
+- Group membership field: Which user field saves the group membership. When selecting "Lookup using a user attribute", this should be a user attribute, otherwise a group attribute.
```

Comment:
For `Group membership field` I am not sure I can easily parse what "_Which user field saves the group membership_." means... I think the phrase "user field" is throwing me, especially since the setting might be a group attribute.
-Does this mean that this setting can accept either the `memberof` user object attribute OR the `___` (if you provided a `Lookup using a user attribute` value)? How can we be more clear about which attributes are possible?
-what exactly does "user field" refer to? 

Suggested wording to be more prescriptive: "Enter the user object attribute or the group object attribute that determines the group membership for each user. If you defined a "Lookup using a user attribute", this should be a user object attribute, otherwise a group object attribute.

Reply:
```suggestion
- Group membership field: The user object attribute or the group object attribute that determines the group membership for a user. If you defined a "Lookup using a user attribute", this should be a user object attribute, otherwise a group object attribute.
```

Would this be fine?